### PR TITLE
Unit test for \Magento\Captcha\Observer\CheckUserForgotPasswordBackendObserver

### PR DESCRIPTION
**Pull request by karyna-t**
### Description (*)
Covers \Magento\Captcha\Observer\CheckUserForgotPasswordBackendObserver with a unit test.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/Captcha/Test/Unit/Observer/CheckUserForgotPasswordBackendObserverTest.php [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+     */
+    private $messageManagerMock;
+
+    /**
+     * @var CheckUserForgotPasswordBackendObserver
+     */
+    private $observer;
+
+    /**
+     * @var MockObject
```
Comment (eduard13) [inline app/code/Magento/Captcha/Test/Unit/Observer/CheckUserForgotPasswordBackendObserverTest.php]: Please set the `CaptchaInterface| MockObject` type here.

File: app/code/Magento/Captcha/Test/Unit/Observer/CheckUserForgotPasswordBackendObserverTest.php [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+use Magento\Captcha\Observer\CheckUserForgotPasswordBackendObserver;
+use Magento\Framework\App\Action\Action;
+use Magento\Framework\App\ActionFlag;
+use Magento\Framework\App\Request\Http as HttpRequest;
+use Magento\Framework\App\Response\Http as HttpResponse;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Message\ManagerInterface;
+use Magento\Framework\Session\SessionManagerInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use PHPUnit\Framework\MockObject\MockObject;
```
Comment (eduard13) [inline app/code/Magento/Captcha/Test/Unit/Observer/CheckUserForgotPasswordBackendObserverTest.php:23]: Please sort ascending the imports.

**Review by eduard13 — CHANGES_REQUESTED** [review]
Hi @karyna-tsymbal-atwix, thank you for your contribution. Could you please check the below comments?
Also, could you please cover one more test case, when the email and/or params aren't provided?
Thank you.

File: app/code/Magento/Captcha/Test/Unit/Observer/CheckUserForgotPasswordBackendObserverTest.php (hunk 1 above)
Comment (karyna-t) [reply app/code/Magento/Captcha/Test/Unit/Observer/CheckUserForgotPasswordBackendObserverTest.php]: thanks, updated

**Review by eduard13 — APPROVED** [review]

**Review by rogyar — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @rogyar, thank you for the review.
[ENGCOM-6900](https://jira.corp.magento.com/browse/ENGCOM-6900) has been created to process this Pull Request
:eight_spoked_asterisk: @rogyar, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Pull request merged**